### PR TITLE
Api models and call feature

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: mentorship/mentor/models.py [hunk 1]
```diff
@@ -32,7 +33,8 @@ class Call(models.Model):
     receiver = models.ForeignKey('Contact', related_name="receiver")
 
     def dial(self):
-        pass
+        api = comms_models.Api.objects.last()
+        api.make_call(self.caller.contact_number, self.receiver.contact_number)
```
Comment (KaitCrawford) [inline mentorship/mentor/models.py]: @miltontony @smn I need help here please. I can't figure out how to get this to call the subclasses make_call() rather than the superclasses make_call() without specifying the subclass which I don't want to do (I want it to be able to use any subclass)

Comment (hodgestar) [inline mentorship/mentor/models.py]: Why do you need to call the subclass' version of `make_call`? (apologies for the drive-by PR reading)

**Comment by KaitCrawford** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
:+1: 
I didn't know about the new celery config stuff... you learn something new everyday..

File: mentorship/mentor/models.py (hunk 1 above)
Comment (KaitCrawford) [inline mentorship/mentor/models.py]: @hodgestar it's because we don't want this code to care what subclass of Api is being used. (All subclasses of Api should define their own make_call() method)
